utils/SummarySpan.md

```diff
@@ -0,0 +1,164 @@
+**trace_option**: There is no difference compared to non-streaming RPC.
+
+#### Case 1: Server Streaming
+**server_latency_ns** = Time elapsed between request received on the server and the
+response status. Response status is sent with the last chunk of the response
+data or after last chunk of the response data.
+
+**lb_latency_ns** = same as above but on LB.
+
+#### Case 2: Client Streaming
+**server_latency_ns** = Time elapsed between last chunk of the request received on
+the server and the response status sent from the server.
+
+**lb_latency_ns** = same as above but on LB.
+
+#### Case 3: Both Streaming
+**server_latency_ns** = Time elapsed between last chunk of the request received on
+the server and the response status sent from the server.
+
+#### Limitations
+There are limitations measuring performance when streaming is used.
+- In case 1, it would be hard to know if the server continued to process the
+request while serving the response. In such case the above latency may not
+provide sufficient information. It only works for the case when the server
+processes the request and has the entire response ready to send it back to the
+client but it sends that in multiple chunks (stream). Here the processing time
+for the request is much larger than sending chunks of data on the wire.
+- In case 2, the server may be able to start processing the request as soon as the
+first chunk is received. We cannot know that without understating the
+application and the content of the request.
+- In case 3, combination of limitation in case 1 and 2 applies here. 
+
+
+### How to interpret the latency measurement?
+
+| client_latency_ns | lb_latency_ns | server_latency_ns | Interpretation |
+|-------------------|---------------|-------------------|----------------|
+| LOW |  LOW |  LOW | Normal. Services are running normally. |
+| HIGH | HIGH | LOW | LB is taking longer to process or network between the LB and the Server is congested. |
+| HIGH | HIGH | HIGH | Server is overloaded or some other underlying issue. Depending on the difference between the latency the problem could spread across. |
+| HIGH | LOW | HIGH | Not possible.  |
+| HIGH | LOW | LOW | There is a Network issue between the client and LB. |
+| HIGH | MISSING | LOW | The issue is not on the server but one cannot conclusively determine if the issue is on LB, or in the network between the LB and the server, or In the network between the Client and the LB |
+| HIGH | MISSING | HIGH | The issue is on the server. Additionally, there could be an issue in other segments. |
+| HIGH | HIGH | MISSING | The issue is on LB or beyond but cannot conclusively determine if it is on the server or not. |
+
+
+### How to use Sampling bit?
+Sampling-bit is used to compare traces and measurements end-to-end. Customers
+can use the sampling-bit information to log traces and measurements on its side.
+The traces and measurement from both sides can then be used to isolate the
+problem.  
```

Comment:
Also mention that it is useful if the Server owner is different than the client owner because then this information helps client to find traces that are sampled on both sides (usually in this case client does not have access to the server traces).

Reply:
updated.